src/node_perf.cc

```diff
@@ -277,7 +265,7 @@ void CreateELDHistogram(const FunctionCallbackInfo<Value>& args) {
       IntervalHistogram::Create(env, interval, [](Histogram& histogram) {
         uint64_t delta = histogram.RecordDelta();
         TRACE_COUNTER1(TRACING_CATEGORY_NODE2(perf, event_loop),
-                        "delay", delta);
+                      "delay", delta);
```

Comment:
Unrelated change
```suggestion
                        "delay", delta);
```

Reply:
@aduh95 
Thanks for the comment.
I thought it would be okay because it was minor, so I edited it together. Removed. 😄